2019/06.md

```diff
@@ -91,6 +91,7 @@ Supporting materials includes slides, a link to the proposal repository, a link
     | 0     | 15m     | [Array.isTemplateObject][] for Stage 1 or 2               | Mike Samuel |
     | 0     | 15m     | [evalable][] for Stage 1 or 2                             | Mike Samuel |
     | 0     | 15m     | [Host compile value adjustment][] for Stage 1 or 2        | Mike Samuel |
+    | 1     | 60m     | [Optional Chaining](https://github.com/tc39/proposal-optional-chaining/) for Stage 2 | Justin Ridgewell |
```

Comment:
Oh, this should be up above the stage 0 presentations (between Intl.DisplayNames and Array.isTemplateObject)